apiary.apib

```diff
@@ -225,3 +225,58 @@ When existing user opens organisation via referral link we need to set (or updat
         {
             "msg": "Referral's inviter was updated."
         }
+        
+### Claim Reward [ /organizations/{id}/referral-system/claim-reward] [GET]         
+
+
++ case if `rewards_type` : `1` for subscription months
+
++ Request (application/json)
```

Comment:
@hakob9315 this is a request for 'prize', not 'subscription'. Please swap requests